src/engraving/layout/layoutpage.cpp

```diff
@@ -403,10 +403,10 @@ void LayoutPage::layoutPage(const LayoutContext& ctx, Page* page, qreal restHeig
         qreal ndist = si->distance() - si->height();        // next taller system
         qreal fill  = ndist - dist;                         // amount by which this system distance exceeds next shorter
         if (fill > 0.0) {
-            qreal totalFill = fill * i;                     // space required to add this amount to all shorter systems
+            qreal totalFill = fill * static_cast<qreal>(i);                     // space required to add this amount to all shorter systems
```

Comment:
Better reduce the amount of spaces, to keep the comment aligned

Reply:
done, thanks